device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransport.java

```diff
@@ -242,7 +242,7 @@ public void onMessageSent(Message message, String deviceId, Throwable e)
             try
             {
                 String messageId = message.getCorrelationId();
```

Comment:
Our variable name is `messageId`, but the method is to get a correlation Id. Should we rename the variable to match?

Reply:
I'm good with this.